data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```diff
@@ -528,6 +528,68 @@
     "description": "A baby snake made entirely out of licorice, possibly the children of the larger Licorice Snakes.",
     "copy-from": "mon_licorice_snake",
     "harvest": "licorice_snake",
-    "upgrades": { "age_grow": 200, "into": "mon_licorice_snake" }
+    "upgrades": { "age_grow": 200, "into": "mon_licorice_snake" },
+    "flags": [ "NO_BREED" ]
+  },
+  {
+    "id": "mon_living_pumpkin",
+    "type": "MONSTER",
+    "name": { "str": "Living Pumpkin" },
+    "description": "This strange creature used to be a pumpkin, now it has taken a humanoid form, it's strange pumpkin skin punctured by hundreds of pumpkin stems, it looks at you with a extremely disturbing look as it lurks forward.",
+    "default_faction": "pumpkin_man",
+    "bodytype": "human",
+    "species": [ "PUMPKIN" ],
+    "volume": "230 ml",
+    "weight": "1000 g",
+    "hp": 65,
+    "speed": 45,
+    "material": [ "veggy" ],
+    "symbol": "p",
+    "color": "yellow",
+    "morale": 105,
+    "melee_skill": 4,
+    "melee_dice": 8,
+    "melee_dice_sides": 1,
+    "dodge": 4,
+    "vision_day": 45,
+    "vision_night": 32,
+    "reproduction": { "baby_monster": "mon_pumpkin_kid", "baby_count": 12, "baby_timer": 325 },
+    "harvest": "pumpkin",
+    "flags": [ "SEES", "STUMBLES", "BASHES" ]
+  },
+  {
+    "id": "mon_pumpkin_kid",
+    "type": "MONSTER",
+    "name": { "str": "Living Pumpkin Kid" },
+    "description": "A baby Living Pumpkin, it looks strange with its short stature, pumpkin skin, and child-like appearance.",
+    "copy-from": "mon_living_pumpkin",
+    "harvest": "pumpkin",
+    "upgrades": { "age_grow": 200, "into": "mon_living_pumpkin" },
+    "flags": [ "NO_BREED" ]
+  },
+  {
+    "id": "mon_cane_hulk",
+    "type": "MONSTER",
+    "name": { "str": "Candy Cane Hulk" },
```

Comment:
Wait the candy canes turn into hulks directly without any intermediate stages? That doesn't work, we need more stages before they grow into hulks.

Plus shouldn't candy canes be more like giant snakes? They're not really humanoid shaped